tests/unit/example.spec.ts

```diff
@@ -7,4 +7,23 @@ describe('HelloWorld.vue', () => {
     const input = wrapper.findAll('[data-test="input"]');
     expect(input.exists()).toBe(true);
   });
+  it('inputの初期値は空です', () => {
+    const wrapper = shallowMount(HelloWorld);
+    const inputValue = wrapper.vm.$data.inputValue;
+    expect(inputValue).toBe('');
+  });
+  it('inputのvalueが変化すれば、バインディングされたデータも共に変化', () => {
+    const wrapper = shallowMount(HelloWorld);
+    const input = wrapper.findAll('[data-test="input"]');
+    input.setValue('Python is the best language');
+    const inputValue = wrapper.vm.$data.inputValue;
```

Comment:
```suggestion
    const result = wrapper.vm.$data.inputValue;
```

ってresult変数に入れると見やすい気がする

Reply:
@tyankatsu0105
変な名前にせずに、素直にresult変数にした方がいいっていうことですか？